packages/react/src/xcss-prop/index.ts

```diff
@@ -208,5 +208,5 @@ export const cx = <TStyles extends [...XCSSProp<any, any>[]]>(
 
   // The output should be a union type of passed in styles. This ensures the call
   // site of xcss prop can raise violations when disallowed styles have been passed.
-  return ac(actualStyles) as TStyles[number];
+  return ax(actualStyles) as TStyles[number];
```

Comment:
I'm worried this doesn't fail tests, not sure how to test this to be honest.  I don't think anything is testing this properly (else it would've been failing before).

Reply:
a setup like that in `packages/react/src/create-strict-api/__tests__/css-func.test.tsx` could work for testing it right?

Reply:
I thought this would've failed in this test before to be frank: https://github.com/atlassian-labs/compiled/blob/39d9a02c8718a439085e40e6fe8eca1bf5274377/packages/react/src/create-strict-api/__tests__/index.test.tsx#L526, but I think because it was passing to the root `<div className={xcss}>` it wasn't a problem, it's when it's wrapped in `<Component xcss={xcss}>` it becomes a problem…?

Added tests in f0f18993 and can confirm, without this `ax` diff it breaks:
<img width="794" alt="Screenshot 2024-07-19 at 1 18 44 PM" src="https://github.com/user-attachments/assets/2f112b54-709a-4a47-a148-c57717aa4ef7">